src/language-server/providers/schema/file.ts

```diff
@@ -3,22 +3,17 @@ import {
   GraphQLSchema,
   buildClientSchema,
   Source,
-  buildSchema,
   printSchema,
   parse,
-  visit,
+  buildASTSchema,
 } from "graphql";
+import { mergeTypeDefs } from "@graphql-tools/merge";
 import { readFileSync } from "fs";
 import { extname, resolve } from "path";
 import { GraphQLSchemaProvider, SchemaChangeUnsubscribeHandler } from "./base";
 import { NotificationHandler } from "vscode-languageserver/node";
 import { Debug } from "../../utilities";
-import { buildSchemaFromSDL } from "apollo-graphql";
```

Comment:
`apollo-graphql` still depends on `graphql` 15, and we only use this one function (source: https://github.com/apollographql/apollo-tooling/blob/master/packages/apollo-graphql/src/schema/buildSchemaFromSDL.ts#L87)

I *hope* my replacement code here is equivalent, but this is what I'm most unsure about in this PR.